src/components/NavBar/TopicDropdown.tsx

```diff
@@ -0,0 +1,61 @@
+import { ReactElement, useState } from 'react';
+import { ChevronDownIcon } from '@heroicons/react/solid';
+import { DropdownList, ItemType } from '../Dropdown';
+
+const topics: ItemType[] = [
+  {
+    id: 'topic-general',
+    label: 'General Science',
+  },
+  {
+    id: 'topic-astrophysics',
+    label: 'Astrophysics',
+  },
+  {
+    id: 'topic-heliophysics',
+    label: 'Heliophysics',
+  },
+  {
+    id: 'topic-planetary',
+    label: 'Planetary Science',
+  },
+  {
+    id: 'topic-earth',
+    label: 'Earth Science',
+  },
+  {
+    id: 'topic-biophysical',
+    label: 'Biological & Physical Science',
+  },
+];
+
+export const TopicDropdown = (): ReactElement => {
+  const [selectedTopic, setSelectedTopic] = useState<ItemType>(topics[0]);
+
+  const getLabelNode = (label: string) => (
+    <div
+      id="topicSelector"
+      className="navbar-bg-color navbar-text-color flex items-center justify-between w-64 border border-gray-50 border-opacity-50 rounded-sm cursor-pointer"
+    >
+      <span className="inline-block align-baseline p-1.5">{label}</span>
+      <ChevronDownIcon className="inline m-1.5 w-4 h-4" />
+    </div>
+  );
+
+  return (
+    <DropdownList
+      label={getLabelNode(selectedTopic.label)}
+      items={topics}
+      onSelect={(topicId: string) => {
```

Comment:
One small nit, instead of these functions inside the props, try to use handlers (i.e. `handleOnSelect`).  When you have a lambda type function like this it gets re-evaluated at every render.

Reply:
Done